model/sparql/GenericSparql.php

```diff
@@ -2243,22 +2242,31 @@ public function listConceptGroupContents($groupClass, $group, $lang,$showDepreca
      * @param string $lang language of labels to return
      * @param int $offset offset of results to retrieve; 0 for beginning of list
      * @param int $limit maximum number of results to return
+     * @param boolean $showDeprecated whether to include deprecated concepts in the change list
      * @return string sparql query
      */
-    private function generateChangeListQuery($prop, $lang, $offset, $limit=200) {
+    private function generateChangeListQuery($prop, $lang, $offset, $limit=200, $showDeprecated=false) {
         $fcl = $this->generateFromClause();
         $offset = ($offset) ? 'OFFSET ' . $offset : '';
 
+        //Additional clauses when deprecated concepts need to be included in the results
+        $deprecatedVars = ($showDeprecated) ? '?replacedBy ?deprecated ?replacingLabel' : '';
+        $deprecatedOptions = ($showDeprecated) ? 'UNION { ?concept dc:isReplacedBy ?replacedBy ; dc:modified ?date2 . BIND(COALESCE(?date2, ?date) AS ?date) OPTIONAL { ?replacedBy skos:prefLabel ?replacingLabel . FILTER (langMatches(lang(?replacingLabel), \''.$lang.'\')) }} OPTIONAL { ?concept owl:deprecated ?deprecated . }' : '';
+
         $query = <<<EOQ
-SELECT DISTINCT ?concept ?date ?label $fcl
-WHERE {
-  ?concept a skos:Concept .
-  ?concept $prop ?date .
-  ?concept skos:prefLabel ?label .
-  FILTER (langMatches(lang(?label), '$lang'))
-}
-ORDER BY DESC(YEAR(?date)) DESC(MONTH(?date)) LCASE(?label)
-LIMIT $limit $offset
+        SELECT ?concept ?date ?label $deprecatedVars $fcl
+        WHERE {
+            ?concept a skos:Concept ;
+	        skos:prefLabel ?label .
+            FILTER (langMatches(lang(?label), '$lang'))
+            {
+                ?concept $prop ?date .
+                MINUS { ?concept owl:deprecated True . }
+            }
+            $deprecatedOptions
+        }
+        ORDER BY DESC(YEAR(?date)) DESC(MONTH(?date)) LCASE(?label) DESC(?concept)
+        LIMIT $limit $offset
 EOQ;
```

Comment:
This changes the indentation of the SPARQL query which has two effects:

1) Wherever it is logged/printed out, this will have extra whitespace in the beginning
2) It complete rewrites the query, thus, git diff here does not show immediately what has changed

Please consider reverting the indentation.